src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -958,6 +989,77 @@ export class ExposureNotificationService {
     return {summary: currentSummary, isNext: false};
   }
 
+  public async saveDisplayExposureHistory() {
+    await this.storageService.save(
+      StorageDirectory.ExposureNotificationServiceDisplayExposureHistoryKey,
+      JSON.stringify(this.displayExposureHistory.get()),
+    );
+  }
+
+  /** this function is only for use on the ExposureHistoryScreen - it only effects the display logic */
```

Comment:
Maybe add - a why this needs to exist after the initial comment

Reply:
done